agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -62,6 +63,25 @@ private static boolean isEmpty(String str) {
     return str == null || str.trim().isEmpty();
   }
 
+  public enum SpanKind {
+    @JsonProperty("server")
+    SERVER(io.opentelemetry.api.trace.SpanKind.SERVER),
```

Comment:
is io.opentelemetry.api.trace.SpanKind an enum? can we reuse it rather than create a new one?  if their spankind is updated, we shouldn't need to update ours.

Reply:
our convention is for json enum values to be lowercase, and we can't add `@JsonProperty` to those classes since they are not under our control. I will add a TODO marker here to investigate other options for mapping external annotations to lowercase.